src/backend/joanie/core/utils/contract.py

```diff
@@ -32,7 +32,7 @@ def _get_base_signature_backend_references(
         extra_filters = {}
 
     base_query = Contract.objects.filter(
-        order__state=enums.ORDER_STATE_VALIDATED,
+        order__state=enums.ORDER_STATE_COMPLETED,
```

Comment:
IMO, organization should be able to sign a contract as soon as a student has signed. Not only if the order has been fully paid. As we are checking the signature state of the contract, I think we can assume to just filter out canceled order.

```suggestion
        ~Q(order__state=enums.ORDER_STATE_CANCELED),
```

Reply:
TODO added.